packages/hooks/src/useControlledValue.tsx

```diff
@@ -0,0 +1,58 @@
+import { ChangeEventHandler, useEffect, useState } from 'react';
```

Comment:
From @TheSonOfThomp SearchResults PR. I added it to my PR so that I can use it.

Reply:
Created a separate PR here https://github.com/mongodb/leafygreen-ui/pull/1603

Reply:
thanks, i'll take a look!